ui/app/AppLayouts/Chat/stores/RootStore.qml

```diff
@@ -655,9 +656,10 @@ QtObject {
         readonly property bool isUserAllowedToSendMessage: {
             if (_d.activeChatType === Constants.chatType.oneToOne && _d.oneToOneChatContact) {
                 return _d.oneToOneChatContact.contactRequestState === Constants.ContactRequestState.Mutual
-            }
-            else if(_d.activeChatType === Constants.chatType.privateGroupChat) {
+            } else if (_d.activeChatType === Constants.chatType.privateGroupChat) {
                 return _d.amIMember
+            } else if (_d.activeChatType === Constants.chatType.communityChat) {
+                return currentChatContentModule().viewAndPostPermissionsSatisfied
```

Comment:
I think the problem is here. This one doesn't check if we are admin.

To be honest, we should probably refactor this code so that the backend changes `viewAndPostPermissionsSatisfied` when being an admin. That would simplify the front-end. 
We can open a new issue and do it during the refactor weeks.

Reply:
Yeah, that would be great to take this into account; I'll have a look

Reply:
OK, updated, I hope the permission check is fine like that; imho better than doing the same in QML